nautilus_trader/cache/cache.pyx

```diff
@@ -1115,6 +1125,11 @@ cdef class Cache(CacheFacade):
         for bar in bars:
             cached_bars.appendleft(bar)
 
+        bar = bars[-1]
+        cdef PriceType price_type = bar.bar_type.spec.price_type
+        if price_type in (PriceType.BID, PriceType.ASK):
```

Comment:
Will be more efficient breaking this out into an `if else` block, rather than the Python tuple thing. 

I can tidy this up though